tests/fixtures/yaml_dicts/index.md

```diff
@@ -0,0 +1,14 @@
+extends: index.liquid
+
+title:   My first Blogpost
+date:    24/08/2014 at 15:36
+some-data:
+  - nested:
+      dicts: Works
+  - nested:
+      dicts: just fine
+---
```

Comment:
I think this is a reasonable feature for some specific use cases.  When a user's data becomes more complex or needs to be shared across files, we'd probably want to implement something like Jekyll's [data files](https://jekyllrb.com/docs/datafiles/) feature.